easybuild/easyconfigs/i/iomkl/iomkl-2015.01.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = "Toolchain"
+
+name = 'iomkl'
+version = '2015.01'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolchain Compiler Edition provides Intel C/C++ and Fortran compilers, Intel MKL & OpenMPI."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+compver = '2015.1.133-GCC-4.9.2'
+comp = ('iccifort', compver)
+ompi = 'OpenMPI'
+ompiver = '1.8.4'
```

Comment:
no need for defining `ompi`, since `OpenMPI` is only used in one place?

```python
dependencies = [
    ...
    ('OpenMPI', '1.8.4', '', comp),
   ...
]
```